### ISSUE TITLE
ArgumentError during build step caused by `npm install`

### ISSUE THREAD

**Issue by minibikini** [labzero/bootleg#169]
Hi, thanks for the great tool!

I had this task in my `deploy.exs`:
```elixir
after_task :compile do
  remote :build do
    "[ -f package.json ] && npm install || true"
    "[ -f brunch-config.js ] && [ -d node_modules ] && ./node_modules/brunch/bin/brunch b -p || true"
    "[ -d deps/phoenix ] && mix phoenix.digest || true"
  end
end
```

Suddenly my builds start failing with this output:
```
...
[my-build-host] │ │ │ │ │ └── supports-color@2.0.0
[my-build-host] │ │ │ │ └─┬ supports-color@3.2.3
[my-build-host] │ │ │ │   └── has-flag@1.0.0
[my-build-host] │ │ │ ├── postcss-modules-extract-imports@1.1.0
[my-build-host] │ │ │ ├─┬ postcss-modules-local-by-default@1.2.0
[my-build-host] │ │ │ │ └─┬ css-selector-tokenizer@0.7.0
[my-build-host] │ │ │ │   ├── cssesc@0.1.0
[my-build-host] │ │ │ │   ├── fastparse@1.1.1
[my-build-host] │ │ │ │   └── regexpu-core@1.0.0
[my-build-host] │ │ │ ├── postcss-modules-scope@1.1.0
[my-build-host] │ │ │ └── postcss-modules-values@1.3.0
[my-build-host] │ │ ├─┬ generic-names@1.0.2
[my-build-host] │ │ │ └─┬ loader-utils@0.2.17
[my-build-host] │ │ │   ├── big.js@3.1.3
[my-build-host] │ │ │   └── emojis-list@2.1.0
[my-build-host] │ │ └── string-hash@1.1.3
[my-build-host] │ └─┬ progeny@0.9.0
[my-build-host] │   ├─┬ chalk@1.1.3
[my-build-host] │   │ ├── ansi-styles@2.2.1
[my-build-host] │   │ └── supports-color@2.0.0
[my-build-host] │   └─┬ fs-mode@1.0.1
[my-build-host] │     └─┬ cbify@1.0.0
** (ArgumentError) argument error
    (stdlib) :io.put_chars(:standard_io, :unicode, [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[...] | "\e[0m"] | "\e[1m"] | "\e[34m"], "[my-build-host] "] | "\e[0m"], "      ├─┬ cliui@2.1.0"], "\n"] | "\e[0m"] | "\e[1m"] | "\e[34m"], "[my-build-host] "] | "\e[0m"], "      │ ├─┬ center-align@0.1.3"], "\n"] | "\e[0m"] | "\e[1m"] | "\e[34m"], "[my-build-host] "] | "\e[0m"], "      │ │ ├─┬ align-text@0.1.4"], "\n"] | "\e[0m"] | "\e[1m"] | "\e[34m"], "[my-build-host] "] | "\e[0m"], "      │ │ │ └── longest@1.0.1"], "\n"] | "\e[0m"] | "\e[1m"] | "\e[34m"], "[my-build-host] "] | "\e[0m"], "      │ │ └── lazy-cache@1.0.4"], "\n"] | "\e[0m"] | "\e[1m"] | "\e[34m"], "[my-build-host] "] | "\e[0m"], "      │ ├── right-align@0.1.3"], "\n"] | "\e[0m"] | "\e[1m"] | "\e[34m"], "[my-build-host] "] | "\e[0m"], "      │ └── wordwrap@0.0.2"] | "\e[0m"], 10])
    lib/bootleg/ssh.ex:90: Bootleg.SSH.capture/3
    lib/sshkit/ssh/channel.ex:224: SSHKit.SSH.Channel.loop/4
    lib/sshkit/ssh.ex:125: SSHKit.SSH.run/3
    lib/bootleg/ssh.ex:70: anonymous fn/2 in Bootleg.SSH.run/2
    (elixir) lib/enum.ex:1255: Enum."-map/2-lists^map/1-0-"/2
    lib/bootleg/ssh.ex:109: Bootleg.SSH.run!/2
    (elixir) lib/enum.ex:1255: Enum."-map/2-lists^map/1-0-"/2
```

I resolved this problem by replacing:
```
"[ -f package.json ] && npm install || true"
```
to
```
"[ -f package.json ] && npm install --silent > /dev/null || true"
```

**Comment by minibikini** [comment]
Looks like this issue related to #161 and #167

**Comment by holetse** [comment]
Thanks for the report! It does look like the same issue as #161. Would it be possible to try the fix from #167 and let us know if it resolves the issue for you?
